### ISSUE TITLE
Prevent footer/pagination on certain pages. 

### ISSUE THREAD

**Issue by sarmadzaki** [puppeteer/puppeteer#4879]
I'm using puppeteer for pdf generation the problem is i want to show footer with pagination on all pages except cover/title page as i have few cover pages in the pdf. Basically i came across a solution for this using `@page {margin-bottom: 100px} @page:first {margin-bottom: 0}`. 
Expected: It should apply the margin bottom on all pages except the first one.
Actual: It applies the margin bottom on all pages.

**Comment by edcoreweb** [comment]
Bumped into this as well, from what I can understand from the chromium source code, it prints each page at a time, so @page:first applies to all pages, as there is only one page present at a time. Would be great if it will allow a function call on each page so you can handle the numbering in user-land.

TL;DR: It's not possible as of now.
